cms/static/js/views/paging_header.js

```diff
@@ -29,100 +29,96 @@ define([
                     messageHtml = this.messageHtml(),
                     isNextDisabled = lastPage === 0 || currentPage === lastPage;
 
-                this.$el.html(_.template(paging_header_template)({ messageHtml: messageHtml}));
-                this.$(".previous-page-link")
-                    .toggleClass("is-disabled", currentPage === 1)
+                this.$el.html(_.template(pagingHeaderTemplate)({ messageHtml: messageHtml}));
```

Comment:
This still needs to be switched to:

    HtmlUtils.setHtml(this.$el, HtmlUtils.template(pagingHeaderTemplate)({messageHtml: messageHtml}));